micro-ui/web/micro-ui-internals/packages/modules/workbench/CHANGELOG.md

```
@@ -1,4 +1,7 @@
 
+## [1.1.1]  [28-October-2025]
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)